### ISSUE TITLE
[feature] Everything is a blog

### ISSUE THREAD

**Issue by Vipermaseg** [mtn/cocoa-eh-hugo-theme#56]
Right now I have a very simple Hugo with two tabs, Blog and Music, but my pals quickly started clomplaining about not being able to comment on the music section. And it is a real shame, I think comments should be available along with navigation buttons, like two blogs.

I thought of just having a list of tab names on the config file pointing to the ones to be treated as blogs.

I think I can do it this weekend, but I wanted to discuss it first, since you might have another vision or another solution.

Cheers.

**Comment by mtn** [comment]
Ah, are you saying comments are only supported on a single site? Neither Alexis (the original maintainer) nor I actually use comments, so I'm not totally familiar. What you described sounds reasonable to me!

**Comment by Vipermaseg** [comment]
Yes, comments are only available on the "blog" tab, which is programmed with some extra stuff.

I've already fixed this on my personal repo, you can check It out, but I haven't made a pull request yet because I didn't properly branch those changes from other changes that I am waiting for you guys to merge t.t

**Comment by mtn** [comment]
Thanks -- if you're able, I'm happy to merge a PR. Otherwise, I'm happy to add it myself, but can only get to this in a few days.

**Comment by mtn** [comment]
Here's a quick solution sketch before I implement anything (I think it's roughly what you had):

`blog/single.html` will be `single.html` for any section in the array `blogs` set in the config file. Also, to avoid breaking existing blogs, either `blog` will be included by default (if the parameter isn't set) or it will be included regardless. Sections not in `blogs` will use the existing `_default/single.html`.

For clarity I'll probably just have `_default/single.html` be a if-else block, with sub-templates.

Let me know if you have any disagreements. If I don't hear anything, I'll go ahead and implement it as is.

**Comment by mtn** [comment]
@Vipermaseg (in case you'll see this)

I have what is probably a fix on on `any_page_comment`. I'm not sure when/if I broke this, but it seemed like no index was being used to render sections that weren't called blog. Not sure what that was about. Regardless, with [this](https://gist.github.com/mtn/035937b945210a148af746fb91811241) config file, we can see that "blog" pages render correctly, and with comments (I don't use them, so for testing I just put a block of text where they'd render)

(on blog)
![screen shot 2018-06-16 at 1 04 58 pm](https://user-images.githubusercontent.com/4515110/41501273-fdadd3be-7166-11e8-8f6d-4bf3341f64c3.png)

(on articles)
![screen shot 2018-06-16 at 1 13 10 pm](https://user-images.githubusercontent.com/4515110/41501280-2bfa3992-7167-11e8-9cf5-89fe2b23756c.png)

And on a section that isn't a blog, they don't render:

![screen shot 2018-06-16 at 1 13 35 pm](https://user-images.githubusercontent.com/4515110/41501281-32255e96-7167-11e8-9655-766ceeb01a4d.png)

This change also modified index, making it so the logo appears the same size on all index pages, etc. I'm not sure if this was broken (I know it worked on some people's blogs, so I'm a bit worried I broke it recently and will test it quickly after this), but now everything that's a landing page uses the same template, as they should.

I'd really appreciate a tester, anyone uses this functionality. Also, as it stands existing blogs will break until the line `blogs = ["blog"]` is added in the config. I'm not sure how I want to deal with this.
